openmc/mgxs/mgxs.py

```diff
@@ -5891,3 +5895,444 @@ def get_units(self, xs_type='macro'):
         else:
             raise ValueError('Unable to return the units of InverseVelocity'
                              ' for xs_type other than "macro"')
+
+
+class SurfaceMGXS(MGXS):
```

Comment:
Thoughts on renaming to `MeshSurfaceMGXS` to be consistent with the nomenclature we use elsewhere for these surface tallies?

Reply:
I think this is the last unresolved conversation. Do you think we should rename it or leave that for a later change ?

Reply:
I'd agree with the renaming. If I end up putting transport GET / discontinuity factors (which are defined on fuel outer surfaces) in mgxs I'll either rename it back or create another class of SurfaceMGXS.